Lib/test/test_importlib/test_util.py

```diff
@@ -522,6 +522,15 @@ def test_find_relative_module_missing_package(self):
             self.assertNotIn(name, sorted(sys.modules))
             self.assertNotIn(fullname, sorted(sys.modules))
 
+    def test_ModuleNotFoundError_raised_for_broken_module_name(self):
```

Comment:
It's not that the name is broken, it's that someone tried to import a submodule on something that isn't a package. So a better name might be `test_find_submodule_in_module(self)`.

Also remove the docstring please (unittest prints that out instead of the method name in verbose mode and we prefer the former to make it easier to find the failing test, plus the formatting doesn't follow PEP 8).